site/content/community/meetings/_index.adoc

```diff
@@ -24,3 +24,52 @@ cascade:
   type: docs
   no_list: false
 ---
+
+== Upcoming Meetings
+
+We have bi-weekly community meeting using Google Meet.
+
+Join the https://groups.google.com/u/1/g/polaris-community-sync[polaris-community-sync Google group].
```

Comment:
nit: I think `Google Group` is the correct styling, especially since we used `Google Meet` above